internal/mode/static/state/graph/httproute.go

```diff
@@ -225,6 +227,11 @@ func validatePathMatch(
 		return field.ErrorList{field.Required(fieldPath.Child("value"), "path value cannot be nil")}
 	}
 
+	if strings.HasPrefix(*path.Value, http.InternalRoutePathPrefix) {
+		msg := fmt.Sprintf("path cannot start with %s", http.InternalRoutePathPrefix)
```

Comment:
do you think it's worth adding why it's not allowed? Maybe
```suggestion
		msg := fmt.Sprintf("path cannot start with %s. This prefix is reserved for internal use", http.InternalRoutePathPrefix)
```